status-im/status-react
# bug #5356 - added snackbar for wallet errors, removed ATT from defaul…

fixes #5356 

### Summary:

A number of our users have (silent) problems fetching prices from cryptocompare api - the effect is that total portfolio value in Wallet is shown as zero even if there is some balance in the account. This PR adds the error message, so that in the case of error users would at least be aware of it, rather than to think that they really have no valuable assets.

This PR also removes ATT from the list of default tokens on ropsten (replaced with HND) and adds a migration to remove it from any persisted user settings. This is to remove the false positive error (unable to get token balance) that would appear in the error area on every upgraded app account.

### Testing notes (optional):

- To properly test the main problem, we'll need to block cryptocompare API and then try to use the app.
- There should *not* be an error saying that token balances are not updated (this was triggered by some ATT leftovers)

### Steps to test:
- Block cryptocompare API
- Open wallet and pull to refresh
- There should be an error like in the screenshot

Also, to check balance update failure, e.g. because of connectivity loss:
- Disable wifi/internet
- Open wallet and pull to refresh
- There should be a different error saying balance couldn't be updated

status: ready
